### PR TITLE
add riak-cs-stanchion test for switch and show

### PR DESCRIPTION
**Pull request by kuenishi**
complementary to #657

### CODE REVIEW

File: riak_test/tests/stanchion_switch_test.erl [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+%% keys for non-multipart objects
+-define(TEST_BUCKET,        "riak-test-bucket").
+-define(KEY_SINGLE_BLOCK,   "riak_test_key1").
+-define(KEY_MULTIPLE_BLOCK, "riak_test_key2").
+
+%% keys for multipart uploaded objects
+-define(KEY_MP_TINY,        "riak_test_mp_tiny").  % single part, single block
+-define(KEY_MP_SMALL,       "riak_test_mp_small"). % single part, multiple blocks
+-define(KEY_MP_LARGE,       "riak_test_mp_large"). % multiple parts
```
Comment (shino) [inline riak_test/tests/stanchion_switch_test.erl]: These macros except 'TEST_BUCKET' are not used.

File: riak_test/tests/stanchion_switch_test.erl [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% ---------------------------------------------------------------------
+
+-module(stanchion_switch_test).
+
+%% @doc `riak_test' module for testing object get behavior.
```
Comment (shino) [inline riak_test/tests/stanchion_switch_test.erl]: Copy & Paste?

File: riak_test/tests/stanchion_switch_test.erl [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      }].
+
+confirm() ->
+    {UserConfig, {RiakNodes, _CSNodes, _Stanchion}} = rtcs:setup(4),
+
+    lists:foreach(fun(RiakNode) ->
+                          N = rt_cs_dev:node_id(RiakNode),
+                          ?assertEqual("Current Stanchion Adderss: http://127.0.0.1:9095\n",
+                                       rtcs:show_stanchion_cs(N))
+                  end, RiakNodes),
```
Comment (shino) [inline riak_test/tests/stanchion_switch_test.erl:48]: Though there is no difference with respect to list of indexes, `_CSNodes` may be better than `RiakNodes`.

File: src/riak_cs_stanchion_console.erl [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                %% {_, _, SSL} = riak_cs_utils:stanchion_data(),
+                %% currently this does not work due to bad path generation of velvet:ping/3
+                %% ok = velvet:ping(Host, NewPort, SSL),
+
+                %% Or, set this configuration to dummy host/port
+                %% to block bucket/user creation/deletion
+                %% in case of multiple stanchion working.
+                ok = application:set_env(riak_cs, stanchion_ip, Host),
+                ok = application:set_env(riak_cs, stanchion_port, NewPort),
+                Msg2 = io_lib:format("Succesfully witched stanchion to ~s:~s: This change is only effective until restart.~n",
```
Comment (shino) [inline src/riak_cs_stanchion_console.erl]: `lager:info` does not need `~n` at the end of line.
With `~n` blank lines will appear in log file.

File: src/riak_cs_stanchion_console.erl [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+show([]) ->
+    ?SAFELY(begin
+                {Host, Port, SSL} = riak_cs_utils:stanchion_data(),
+                Scheme = case SSL of
+                             true -> "https://";
+                             false -> "http://"
+                         end,
+                io:format("Current Stanchion Adderss: ~s~s:~s~n",
+                          [Scheme, Host, integer_to_list(Port)])
+            end, "Retrieving Stanchion info failed").
```
Comment (shino) [inline src/riak_cs_stanchion_console.erl]: "failed" is duplicated, one in this line and another in `SAFELY` macro.

File: riak_test/tests/stanchion_switch_test.erl (hunk 3 above)
Comment (shino) [inline riak_test/tests/stanchion_switch_test.erl:48]: My bad, sorry. There seems no simple function to change CSNode to id.

File: riak_test/tests/stanchion_switch_test.erl (hunk 1 above)
Comment (kuenishi) [inline riak_test/tests/stanchion_switch_test.erl]: Yeah these are not used. meh.

File: riak_test/tests/stanchion_switch_test.erl (hunk 2 above)
Comment (kuenishi) [inline riak_test/tests/stanchion_switch_test.erl]: Yes. Comments in the code cannot enter into my eyes. :bomb:

File: riak_test/tests/stanchion_switch_test.erl (hunk 3 above)
Comment (kuenishi) [inline riak_test/tests/stanchion_switch_test.erl:48]: Yes, we're free-riding on the riak-test features.

File: src/riak_cs_stanchion_console.erl (hunk 4 above)
Comment (kuenishi) [inline src/riak_cs_stanchion_console.erl]: Sorry, that was the copy and paste.

Comment (shino) [inline src/riak_cs_stanchion_console.erl]: Found tiny typo. `s/witched/switched/` ?

**Comment by kuenishi** [comment]
Fixed, ready again!

**Comment by shino** [comment]
:+1:

**Pull request merged**